### PR TITLE
Add name of the non-existing project to error-message

### PR DESCRIPTION
**Pull request by tobias74**
This PR should improve debugging angular.json. In my case I could not figure out which project was missing. There was a small typo in one of the references to the project-name. Displaying the name of the missing project helps finding the error.

### CODE REVIEW

**Comment by tobias74** [comment]
There is a discussion concerning this issue on SO: [https://stackoverflow.com/questions/57585781/angular-ng-serve-command-throws-error-an-unhandled-exception-occurred-project](https://stackoverflow.com/questions/57585781/angular-ng-serve-command-throws-error-an-unhandled-exception-occurred-project)

**Comment by tobias74** [comment]
as it seems my commit-message is wrong... how do I fix that?

File: packages/angular_devkit/architect/node/node-modules-architect-host.ts [hunk 1]
```diff
@@ -129,7 +129,7 @@ export class WorkspaceNodeModulesArchitectHost implements ArchitectHost<NodeModu
 
     const projectDefinition = this._workspace.projects.get(projectName);
     if (!projectDefinition) {
-      throw new Error('Project does not exist.');
+      throw new Error('Project does not exist: ' + projectName);
```
Comment (alan-agius4) [inline packages/angular_devkit/architect/node/node-modules-architect-host.ts:132]: Can we change this to `Project "${projectName}" does not exist.` please?

Comment (tobias74) [reply packages/angular_devkit/architect/node/node-modules-architect-host.ts]: done :-)

**Comment by tobias74** [comment]
@googlebot I fixed it.

**Comment by tobias74** [comment]
The tests have failed with 'Error: Running "npm install" returned error code'.

How can I re-schedule the tests?

**Comment by tobias74** [comment]
I got stuck rebasing and squashing the commits, so to simplify matters, I just opend a new and cleaned-up PR: https://github.com/angular/angular-cli/pull/18502

I will close this one in favour of the new PR

**Pull request closed without merging**